pkg/lib/bundle/validate.go

```diff
@@ -0,0 +1,256 @@
+package bundle
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	FileStoreDir     = "bundle-"
+	BundleTarFile    = "bundle.tar"
+	ManifestJsonFile = "manifest.json"
+)
+
+type ManifestJson struct {
+	Config string   `json:"Config"`
+	Layers []string `json:"Layers"`
+}
+
+type LabelsJson struct {
+	Config ConfigData `json:"config"`
+}
+
+type ConfigData struct {
+	Labels map[string]string `json:"Labels"`
+}
+
+// ValidateFunc is used to validate bundle container image.
+// Inputs:
+// @imageTag: The image tag that is applied to the bundle image
+// @imageBuilder: The image builder tool that is used to build container image
+// (docker or podman)
+func ValidateFunc(imageTag, imageBuilder string) error {
+	dir, err := ioutil.TempDir("", FileStoreDir)
+	log.Infof("Create a temp directory at %s", dir)
```

Comment:
We should consider using a global `log.Logger` here and then providing an exported function that allows setting that logger.

Since this code is primarily meant to be used as a library, this would give callers more flexibility in how the logs are consumed.